infrastructure/messenger-db-migrator-deploy.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
+
+# This configuration requires a new commit to change
+NODE_ENV=production
+PORT=4000
+JSON_BODY_LIMIT=20b
```

Comment:
I think this limit is much too low. The [Express documentation](https://expressjs.com/en/api.html#express.json) shows a default of `100kb` but perhaps we should discuss this as a team and decide on a limit that works for our service.